Auto update failing for miniserve

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the miniserve package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating miniserve [Current version: 0.21.0]
INFO: package being updated to 0.22.0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  146k    0  146k    0     0   337k      0 --:--:-- --:--:-- --:--:--  337k

You are about to commit these changes:

--------------------
diff --git a/packages/miniserve/build.sh b/packages/miniserve/build.sh
index 3c87ca246..2dae7bb76 100644
--- a/packages/miniserve/build.sh
+++ b/packages/miniserve/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/svenstaro/miniserve
 TERMUX_PKG_DESCRIPTION="Tool to serve files and dirs over HTTP"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.21.0"
+TERMUX_PKG_VERSION="0.22.0"
 TERMUX_PKG_SRCURL=https://github.com/svenstaro/miniserve/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=8816d8379321fd3160f57d771b4b743f04bab57dd1ec8d04def0b15a96bde87c
+TERMUX_PKG_SHA256=325f6cde391c468000b1bdcc8455ec2c6950b3c930029187671c536507b185ba
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS=libbz2
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

miniserve: update to 0.22.0


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
Creating new container...
Unable to find image 'ghcr.io/termux/package-builder:latest' locally
latest: Pulling from termux/package-builder
2b55860d4c66: Pulling fs layer
6e508719b9e2: Pulling fs layer
4dc23316705c: Pulling fs layer
76781c0d619f: Pulling fs layer
5f5905e175c0: Pulling fs layer
023148d75e97: Pulling fs layer
76781c0d619f: Waiting
5f5905e175c0: Waiting
023148d75e97: Waiting
6e508719b9e2: Verifying Checksum
6e508719b9e2: Download complete
4dc23316705c: Download complete
2b55860d4c66: Verifying Checksum
2b55860d4c66: Download complete
76781c0d619f: Verifying Checksum
76781c0d619f: Download complete
023148d75e97: Verifying Checksum
023148d75e97: Download complete
2b55860d4c66: Pull complete
6e508719b9e2: Pull complete
4dc23316705c: Pull complete
76781c0d619f: Pull complete
5f5905e175c0: Verifying Checksum
5f5905e175c0: Download complete
5f5905e175c0: Pull complete
023148d75e97: Pull complete
Digest: sha256:9d1a93b1c5054ecf2b11a453edfdc94965d15e6752d354edc89f16bac17b4ae1
Status: Downloaded newer image for ghcr.io/termux/package-builder:latest
8bfdaea6c4983c4faef9cc2a935a1fda908a22e4046d7eca227e790c02d32cb5
gpg: key B0076E490B71616B: 17 signatures not checked due to missing keys
gpg: key B0076E490B71616B: public key "Henrik Grimler <henrik@grimler.se>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no ultimately trusted keys found
gpg (GnuPG) 2.2.27; Copyright (C) 2021 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

gpg: key 5A897D96E57CF20C: public key "Termux Releases (Termux automatic builds) <contact@termux.dev>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   1  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: depth: 1  valid:   1  signed:   0  trust: 1-, 0q, 0n, 0m, 0f, 0u
gpg: next trustdb check due at 2024-08-25
gpg (GnuPG) 2.2.27; Copyright (C) 2021 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

termux - building miniserve for arch aarch64...
Downloading https://github.com/termux/termux-elf-cleaner/releases/download/v2.1.1/termux-elf-cleaner
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 63912  100 63912    0     0   147k      0 --:--:-- --:--:-- --:--:--  147k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  63541      0 --:--:-- --:--:-- --:--:-- 63757
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2902      0 --:--:-- --:--:-- --:--:--  2902
100   833  100   833    0     0   2901      0 --:--:-- --:--:-- --:--:--  2902
gpg: Signature made Tue 20 Sep 2022 06:24:10 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: checking the trustdb
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   2  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 2u
gpg: next trustdb check due at 2024-08-25
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1385k  100 1385k    0     0  2768k      0 --:--:-- --:--:-- --:--:-- 2771k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  45653      0 --:--:-- --:--:-- --:--:-- 45769
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3142      0 --:--:-- --:--:-- --:--:--  3143
gpg: Signature made Sun 04 Sep 2022 06:32:25 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/stable/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 49200  100 49200    0     0   154k      0 --:--:-- --:--:-- --:--:--  154k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  52321      0 --:--:-- --:--:-- --:--:-- 52480
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3587      0 --:--:-- --:--:-- --:--:--  3575
gpg: Signature made Mon 19 Sep 2022 07:10:44 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  347k  100  347k    0     0   967k      0 --:--:-- --:--:-- --:--:--  968k
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libb/libbz2/libbz2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 26634  100 26634    0     0   130k      0 --:--:-- --:--:-- --:--:--  130k
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bzip2/bzip2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 25506  100 25506    0     0   121k      0 --:--:-- --:--:-- --:--:--  122k
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/svenstaro/miniserve/archive/v0.22.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  146k    0  146k    0     0   234k      0 --:--:-- --:--:-- --:--:--  234k
Applying ndk-patch: bits-struct_file.h.patch
Applying ndk-patch: dirent.h.patch
Applying ndk-patch: grp.h.patch
Applying ndk-patch: langinfo.h.patch
Applying ndk-patch: linux-fcntl.h.patch
Applying ndk-patch: paths.h.patch
Applying ndk-patch: pwd.h.patch
Applying ndk-patch: redefine-TCSAFLUSH.patch
Applying ndk-patch: semaphore.h.patch
Applying ndk-patch: stdio.h.patch
Applying ndk-patch: stdlib.h.patch
Applying ndk-patch: sys-cdefs.h.patch
Applying ndk-patch: syslog.patch
Applying ndk-patch: unistd.h.patch
Applying ndk-patch: utmp.h.patch
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libz.so'
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
info: syncing channel updates for '1.63.0-x86_64-unknown-linux-gnu'
info: latest update on 2022-08-11, rust version 1.63.0 (4b91a6ea7 2022-08-08)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'
info: default toolchain set to '1.63.0-x86_64-unknown-linux-gnu'

  1.63.0-x86_64-unknown-linux-gnu installed - rustc 1.63.0 (4b91a6ea7 2022-08-08)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: installing component 'rust-std' for 'aarch64-linux-android'
  Installing miniserve v0.22.0 (/home/builder/.termux-build/miniserve/src)
    Updating crates.io index
 Downloading crates ...
  Downloaded actix-codec v0.5.0
  Downloaded block-buffer v0.10.3
  Downloaded alloc-stdlib v0.2.2
  Downloaded actix-web-codegen v4.1.0
  Downloaded actix-rt v2.7.0
  Downloaded actix-router v0.5.1
  Downloaded actix-server v2.1.1
  Downloaded unicode-ident v1.0.4
  Downloaded rle-decode-fast v1.0.3
  Downloaded roff v0.2.1
  Downloaded plist v1.3.1
  Downloaded ucd-trie v0.1.5
  Downloaded terminal_size v0.2.1
  Downloaded simplelog v0.12.0
  Downloaded sha1 v0.10.5
  Downloaded rustls-pemfile v1.0.1
  Downloaded gcc v0.3.55
  Downloaded port_check v0.1.5
  Downloaded fast_qr v0.5.1
  Downloaded get_if_addrs v0.5.3
  Downloaded clap v3.2.22
  Downloaded actix-service v2.0.2
  Downloaded tinyvec_macros v0.1.0
  Downloaded unchecked-index v0.2.2
  Downloaded shell-words v1.1.0
  Downloaded proc-macro-error v1.0.4
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded pkg-config v0.3.25
  Downloaded pin-project-lite v0.2.9
  Downloaded num-traits v0.2.15
  Downloaded scopeguard v1.1.0
  Downloaded safemem v0.3.3
  Downloaded siphasher v0.3.10
  Downloaded rand_core v0.6.4
  Downloaded strsim v0.8.0
  Downloaded signal-hook-registry v1.4.0
  Downloaded serde_urlencoded v0.7.1
  Downloaded rustc_version v0.4.0
  Downloaded semver v1.0.14
  Downloaded spin v0.5.2
  Downloaded textwrap v0.11.0
  Downloaded smallvec v1.9.0
  Downloaded time v0.1.44
  Downloaded strsim v0.10.0
  Downloaded termcolor v1.1.3
  Downloaded strum_macros v0.24.3
  Downloaded tokio-rustls v0.23.4
  Downloaded serde v1.0.144
  Downloaded tinyvec v1.6.0
  Downloaded textwrap v0.15.1
  Downloaded unicode-bidi v0.3.8
  Downloaded tracing-core v0.1.29
  Downloaded tracing v0.1.36
  Downloaded tar v0.4.38
  Downloaded webpki-roots v0.22.4
  Downloaded time v0.3.14
  Downloaded serde_json v1.0.85
  Downloaded futures-util v0.3.24
  Downloaded rustls v0.20.6
  Downloaded url v2.3.1
  Downloaded zstd v0.11.2+zstd.1.5.2
  Downloaded syn v1.0.100
  Downloaded unicode_categories v0.1.1
  Downloaded sct v0.7.0
  Downloaded socket2 v0.4.7
  Downloaded thiserror v1.0.35
  Downloaded percent-encoding v2.2.0
  Downloaded tokio-util v0.7.4
  Downloaded alphanumeric-sort v1.4.4
  Downloaded untrusted v0.7.1
  Downloaded version_check v0.9.4
  Downloaded yansi v0.5.1
  Downloaded yaml-rust v0.4.5
  Downloaded phf_shared v0.9.0
  Downloaded maud_macros v0.23.0
  Downloaded proc-macro2 v1.0.43
  Downloaded xattr v0.2.3
  Downloaded webpki v0.22.0
  Downloaded unicode-normalization v0.1.22
  Downloaded tokio v1.21.1
  Downloaded syntect v4.6.0
  Downloaded libc v0.2.133
  Downloaded phf_macros v0.9.0
  Downloaded comrak v0.14.0
  Downloaded xml-rs v0.8.4
  Downloaded linux-raw-sys v0.0.46
  Downloaded encoding_rs v0.8.31
  Downloaded zip v0.6.2
  Downloaded zstd-safe v5.0.2+zstd.1.5.2
  Downloaded onig_sys v69.8.1
  Downloaded num_threads v0.1.6
  Downloaded onig v6.4.0
  Downloaded actix-macros v0.2.3
  Downloaded actix-http v3.2.2
  Downloaded pest v2.3.1
  Downloaded zstd-sys v2.0.1+zstd.1.5.2
  Downloaded ring v0.16.20
  Downloaded phf_generator v0.9.1
  Downloaded actix-files v0.6.2
  Downloaded num_cpus v1.13.1
  Downloaded brotli v3.3.4
  Downloaded xdg v2.4.1
  Downloaded phf v0.9.0
  Downloaded pest_generator v2.3.1
  Downloaded os_str_bytes v6.3.0
  Downloaded codemap v0.1.3
  Downloaded regex-syntax v0.6.27
  Downloaded walkdir v2.3.2
  Downloaded unicode-width v0.1.10
  Downloaded maud v0.23.0
  Downloaded thiserror-impl v1.0.35
  Downloaded num-integer v0.1.45
  Downloaded brotli-decompressor v2.3.2
  Downloaded vec_map v0.8.2
  Downloaded rand v0.8.5
  Downloaded num-rational v0.4.1
  Downloaded entities v1.0.1
  Downloaded chrono-humanize v0.2.2
  Downloaded actix-web-httpauth v0.8.0
  Downloaded get_if_addrs-sys v0.1.1
  Downloaded actix-multipart v0.4.0
  Downloaded parking_lot v0.12.1
  Downloaded futures-task v0.3.24
  Downloaded slab v0.4.7
  Downloaded derive_more v0.99.17
  Downloaded once_cell v1.14.0
  Downloaded mime_guess v2.0.4
  Downloaded num-bigint v0.4.3
  Downloaded digest v0.10.5
  Downloaded futures-io v0.3.24
  Downloaded bit-vec v0.6.3
  Downloaded hex v0.4.3
  Downloaded regex v1.6.0
  Downloaded idna v0.3.0
  Downloaded chrono v0.4.22
  Downloaded clap v2.34.0
  Downloaded alloc-no-stdlib v2.0.4
  Downloaded actix-web v4.2.1
  Downloaded paste v1.0.9
  Downloaded miniz_oxide v0.5.4
  Downloaded hashbrown v0.12.3
  Downloaded mio v0.8.4
  Downloaded hashbrown v0.11.2
  Downloaded twoway v0.2.2
  Downloaded nanoid v0.4.0
  Downloaded memchr v2.5.0
  Downloaded lock_api v0.4.8
  Downloaded lasso v0.5.1
  Downloaded log v0.4.17
  Downloaded fancy-regex v0.7.1
  Downloaded httparse v1.8.0
  Downloaded flate2 v1.0.24
  Downloaded aho-corasick v0.7.19
  Downloaded mime v0.3.16
  Downloaded indexmap v1.9.1
  Downloaded h2 v0.3.14
  Downloaded unicase v2.6.0
  Downloaded typenum v1.15.0
  Downloaded strum v0.24.1
  Downloaded futures-channel v0.3.24
  Downloaded io-lifetimes v0.7.3
  Downloaded linked-hash-map v0.5.6
  Downloaded line-wrap v0.1.1
  Downloaded clap_derive v3.2.18
  Downloaded actix-tls v3.0.3
  Downloaded language-tags v0.3.2
  Downloaded time-macros v0.2.4
  Downloaded libflate v1.2.0
  Downloaded pest_meta v2.3.1
  Downloaded pest_derive v2.3.1
  Downloaded libflate_lz77 v1.1.0
  Downloaded http-range v0.1.5
  Downloaded clap_mangen v0.1.11
  Downloaded c_linked_list v1.1.1
  Downloaded serde_derive v1.0.144
  Downloaded fnv v1.0.7
  Downloaded beef v0.5.2
  Downloaded android_system_properties v0.1.5
  Downloaded jobserver v0.1.24
  Downloaded iana-time-zone v0.1.48
  Downloaded local-waker v0.1.3
  Downloaded heck v0.4.0
  Downloaded futures-macro v0.3.24
  Downloaded futures v0.3.24
  Downloaded local-channel v0.1.3
  Downloaded getrandom v0.2.7
  Downloaded generic-array v0.14.6
  Downloaded lazycell v1.3.0
  Downloaded lazy_static v1.4.0
  Downloaded itoa v1.0.3
  Downloaded futures-executor v0.3.24
  Downloaded bytes v1.2.1
  Downloaded ryu v1.0.11
  Downloaded http v0.2.8
  Downloaded form_urlencoded v1.1.0
  Downloaded dirs-sys v0.3.7
  Downloaded crc32fast v1.3.2
  Downloaded base64 v0.13.0
  Downloaded rand_chacha v0.3.1
  Downloaded cc v1.0.73
  Downloaded futures-sink v0.3.24
  Downloaded futures-core v0.3.24
  Downloaded filetime v0.2.17
  Downloaded convert_case v0.4.0
  Downloaded sha2 v0.10.6
  Downloaded errno v0.2.8
  Downloaded crypto-common v0.1.6
  Downloaded cookie v0.16.0
  Downloaded cpufeatures v0.2.5
  Downloaded bincode v1.3.3
  Downloaded dirs v4.0.0
  Downloaded bitflags v1.3.2
  Downloaded ahash v0.7.6
  Downloaded rustversion v1.0.9
  Downloaded quote v1.0.21
  Downloaded bytestring v1.1.0
  Downloaded cfg-if v1.0.0
  Downloaded byteorder v1.4.3
  Downloaded same-file v1.0.6
  Downloaded proc-macro-hack v0.5.19
  Downloaded clap_complete v3.2.5
  Downloaded bytesize v1.1.0
  Downloaded anyhow v1.0.65
  Downloaded ansi_term v0.12.1
  Downloaded autocfg v1.1.0
  Downloaded bit-set v0.5.3
  Downloaded askama_escape v0.10.3
  Downloaded actix-utils v3.0.0
  Downloaded atty v0.2.14
  Downloaded adler v1.0.2
  Downloaded parking_lot_core v0.9.3
  Downloaded clap_lex v0.2.4
  Downloaded adler32 v1.2.0
  Downloaded ppv-lite86 v0.2.16
  Downloaded pin-utils v0.1.0
  Downloaded httpdate v1.0.2
  Downloaded grass v0.11.2
  Downloaded typed-arena v1.7.0
  Downloaded rustix v0.35.9
   Compiling libc v0.2.133
   Compiling autocfg v1.1.0
   Compiling proc-macro2 v1.0.43
   Compiling quote v1.0.21
   Compiling unicode-ident v1.0.4
   Compiling syn v1.0.100
   Compiling version_check v0.9.4
   Compiling cfg-if v1.0.0
   Compiling once_cell v1.14.0
   Compiling memchr v2.5.0
   Compiling pin-project-lite v0.2.9
   Compiling log v0.4.17
   Compiling futures-core v0.3.24
   Compiling bitflags v1.3.2
   Compiling bytes v1.2.1
   Compiling itoa v1.0.3
   Compiling parking_lot_core v0.9.3
   Compiling scopeguard v1.1.0
   Compiling smallvec v1.9.0
   Compiling futures-sink v0.3.24
   Compiling serde_derive v1.0.144
   Compiling futures-channel v0.3.24
   Compiling hashbrown v0.12.3
   Compiling typenum v1.15.0
   Compiling serde v1.0.144
   Compiling futures-task v0.3.24
   Compiling futures-util v0.3.24
   Compiling crc32fast v1.3.2
   Compiling spin v0.5.2
   Compiling untrusted v0.7.1
   Compiling futures-io v0.3.24
   Compiling pin-utils v0.1.0
   Compiling fnv v1.0.7
   Compiling thiserror v1.0.35
   Compiling num_threads v0.1.6
   Compiling time-macros v0.2.4
   Compiling adler v1.0.2
   Compiling rustls v0.20.6
   Compiling local-waker v0.1.3
   Compiling percent-encoding v2.2.0
   Compiling base64 v0.13.0
   Compiling regex-syntax v0.6.27
   Compiling io-lifetimes v0.7.3
   Compiling ryu v1.0.11
   Compiling ppv-lite86 v0.2.16
   Compiling alloc-no-stdlib v2.0.4
   Compiling paste v1.0.9
   Compiling tinyvec_macros v0.1.0
   Compiling serde_json v1.0.85
   Compiling zstd-safe v5.0.2+zstd.1.5.2
   Compiling siphasher v0.3.10
   Compiling encoding_rs v0.8.31
   Compiling proc-macro-hack v0.5.19
   Compiling rustix v0.35.9
   Compiling httparse v1.8.0
   Compiling tracing-core v0.1.29
   Compiling pkg-config v0.3.25
   Compiling unicode-bidi v0.3.8
   Compiling linux-raw-sys v0.0.46
   Compiling mime v0.3.16
   Compiling convert_case v0.4.0
   Compiling unicode-width v0.1.10
   Compiling ucd-trie v0.1.5
   Compiling heck v0.4.0
   Compiling httpdate v1.0.2
   Compiling language-tags v0.3.2
   Compiling ansi_term v0.12.1
   Compiling strsim v0.8.0
   Compiling safemem v0.3.3
   Compiling vec_map v0.8.2
   Compiling gcc v0.3.55
   Compiling os_str_bytes v6.3.0
   Compiling termcolor v1.1.3
   Compiling rustversion v1.0.9
   Compiling same-file v1.0.6
   Compiling beef v0.5.2
   Compiling strsim v0.10.0
   Compiling codemap v0.1.3
   Compiling xml-rs v0.8.4
   Compiling lazycell v1.3.0
   Compiling lazy_static v1.4.0
   Compiling rle-decode-fast v1.0.3
   Compiling anyhow v1.0.65
   Compiling unchecked-index v0.2.2
   Compiling entities v1.0.1
   Compiling typed-arena v1.7.0
   Compiling http-range v0.1.5
   Compiling unicode_categories v0.1.1
   Compiling shell-words v1.1.0
   Compiling c_linked_list v1.1.1
   Compiling roff v0.2.1
   Compiling byteorder v1.4.3
   Compiling askama_escape v0.10.3
   Compiling adler32 v1.2.0
   Compiling bytesize v1.1.0
   Compiling yansi v0.5.1
   Compiling hex v0.4.3
   Compiling fast_qr v0.5.1
   Compiling strum v0.24.1
   Compiling port_check v0.1.5
   Compiling alphanumeric-sort v1.4.4
   Compiling lock_api v0.4.8
   Compiling tokio v1.21.1
   Compiling indexmap v1.9.1
   Compiling slab v0.4.7
   Compiling num-traits v0.2.15
   Compiling num-integer v0.1.45
   Compiling num-bigint v0.4.3
   Compiling num-rational v0.4.1
   Compiling generic-array v0.14.6
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling unicase v2.6.0
   Compiling ahash v0.7.6
   Compiling cookie v0.16.0
   Compiling bytestring v1.1.0
   Compiling http v0.2.8
   Compiling miniz_oxide v0.5.4
   Compiling actix-utils v3.0.0
   Compiling form_urlencoded v1.1.0
   Compiling rustls-pemfile v1.0.1
   Compiling alloc-stdlib v0.2.2
   Compiling tinyvec v1.6.0
   Compiling phf_shared v0.9.0
   Compiling textwrap v0.11.0
   Compiling line-wrap v0.1.1
   Compiling clap_lex v0.2.4
   Compiling get_if_addrs-sys v0.1.1
   Compiling walkdir v2.3.2
   Compiling libflate_lz77 v1.1.0
   Compiling brotli-decompressor v2.3.2
   Compiling unicode-normalization v0.1.22
   Compiling signal-hook-registry v1.4.0
   Compiling socket2 v0.4.7
   Compiling getrandom v0.2.7
   Compiling time v0.3.14
   Compiling cpufeatures v0.2.5
   Compiling errno v0.2.8
   Compiling num_cpus v1.13.1
   Compiling atty v0.2.14
   Compiling dirs-sys v0.3.7
   Compiling android_system_properties v0.1.5
   Compiling time v0.1.44
   Compiling xattr v0.2.3
   Compiling filetime v0.2.17
   Compiling aho-corasick v0.7.19
   Compiling twoway v0.2.2
   Compiling jobserver v0.1.24
   Compiling actix-service v2.0.2
   Compiling mio v0.8.4
   Compiling tracing v0.1.36
   Compiling flate2 v1.0.24
   Compiling zip v0.6.2
   Compiling libflate v1.2.0
   Compiling brotli v3.3.4
   Compiling idna v0.3.0
   Compiling parking_lot v0.12.1
   Compiling rand_core v0.6.4
error[E0308]: arguments to this function are incorrect
    --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/rustix-0.35.9/src/backend/libc/fs/makedev.rs:15:14
     |
15   |     unsafe { c::makedev(maj as i32, min as i32) }
     |              ^^^^^^^^^^ ----------  ---------- expected `u32`, found `i32`
     |                         |
     |                         expected `u32`, found `i32`
     |
note: function defined here
    --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.133/src/unix/linux_like/android/mod.rs:2794:20
     |
2794 |     pub {const} fn makedev(ma: ::c_uint, mi: ::c_uint) -> ::dev_t {
     |                    ^^^^^^^
help: you can convert an `i32` to a `u32` and panic if the converted value doesn't fit
     |
15   |     unsafe { c::makedev((maj as i32).try_into().unwrap(), min as i32) }
     |                         +          +++++++++++++++++++++
help: you can convert an `i32` to a `u32` and panic if the converted value doesn't fit
     |
15   |     unsafe { c::makedev(maj as i32, (min as i32).try_into().unwrap()) }
     |                                     +          +++++++++++++++++++++

For more information about this error, try `rustc --explain E0308`.
error: could not compile `rustix` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `miniserve v0.22.0 (/home/builder/.termux-build/miniserve/src)`, intermediate artifacts can be found at `/home/builder/.termux-build/miniserve/src/target`
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2022-09-20 06:46:51 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/3087878043">3087878043</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/miniserve/0001-upstream-fix-libc-removing-unsafe-on-makedev.diff
+++ b/packages/miniserve/0001-upstream-fix-libc-removing-unsafe-on-makedev.diff
@@ -1,0 +1,71 @@
+From d96e9957e5725bb67e8ba617cb8bfe087a271ae0 Mon Sep 17 00:00:00 2001
+From: ETKNeil <89385146+ETKNeil@users.noreply.github.com>
+Date: Tue, 20 Sep 2022 13:39:18 +0200
+Subject: [PATCH] Fix libc removing unsafe on makedev
+
+---
+ Cargo.toml                     | 6 +++---
+ src/backend/libc/fs/makedev.rs | 6 +++---
+ 2 files changed, 6 insertions(+), 6 deletions(-)
+
+# diff --git a/Cargo.toml b/Cargo.toml
+# index d9735cb8..dcde3702 100644
+# --- a/Cargo.toml
+# +++ b/Cargo.toml
+# @@ -40,7 +40,7 @@ once_cell = { version = "1.5.2", optional = true }
+#  [target.'cfg(all(not(rustix_use_libc), not(miri), target_os = "linux", any(target_arch = "x86", all(target_arch = "x86_64", target_pointer_width = "64"), all(target_endian = "little", any(target_arch = "arm", all(target_arch = "aarch64", target_pointer_width = "64"), target_arch = "powerpc64", target_arch = "riscv64", target_arch = "mips", target_arch = "mips64")))))'.dependencies]
+#  linux-raw-sys = { version = "0.0.46", default-features = false, features = ["general", "errno", "ioctl", "no_std"] }
+#  libc_errno = { package = "errno", version = "0.2.8", default-features = false, optional = true }
+# -libc = { version = "0.2.126", features = ["extra_traits"], optional = true }
+# +libc = { version = "0.2.133", features = ["extra_traits"], optional = true }
+ 
+#  # Dependencies for platforms where only libc is supported:
+#  #
+# @@ -48,7 +48,7 @@ libc = { version = "0.2.126", features = ["extra_traits"], optional = true }
+#  # backend, so enable its dependencies unconditionally.
+#  [target.'cfg(any(rustix_use_libc, miri, not(all(target_os = "linux", any(target_arch = "x86", all(target_arch = "x86_64", target_pointer_width = "64"), all(target_endian = "little", any(target_arch = "arm", all(target_arch = "aarch64", target_pointer_width = "64"), target_arch = "powerpc64", target_arch = "riscv64", target_arch = "mips", target_arch = "mips64")))))))'.dependencies]
+#  libc_errno = { package = "errno", version = "0.2.8", default-features = false }
+# -libc = { version = "0.2.126", features = ["extra_traits"] }
+# +libc = { version = "0.2.133", features = ["extra_traits"] }
+ 
+#  # Additional dependencies for Linux with the libc backend:
+#  #
+# @@ -69,7 +69,7 @@ features = [
+ 
+#  [dev-dependencies]
+#  tempfile = "3.2.0"
+# -libc = "0.2.126"
+# +libc = "0.2.133"
+#  libc_errno = { package = "errno", version = "0.2.8", default-features = false }
+#  io-lifetimes = { version = "1.0.0-rc1", default-features = false }
+#  # Don't upgrade to serial_test 0.7 for now because it depends on a
+diff --git a/src/backend/libc/fs/makedev.rs b/src/backend/libc/fs/makedev.rs
+index d9089e7f..08ecd872 100644
+--- a/src/backend/libc/fs/makedev.rs
++++ b/src/backend/libc/fs/makedev.rs
+@@ -5,14 +5,14 @@ use crate::fs::Dev;
+ #[cfg(not(any(target_os = "android", target_os = "emscripten")))]
+ #[inline]
+ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+-    unsafe { c::makedev(maj, min) }
++    c::makedev(maj, min)
+ }
+ 
+ #[cfg(all(target_os = "android", not(target_pointer_width = "32")))]
+ #[inline]
+ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+     // Android's `makedev` oddly has signed argument types.
+-    unsafe { c::makedev(maj as i32, min as i32) }
++    c::makedev(maj, min)
+ }
+ 
+ #[cfg(all(target_os = "android", target_pointer_width = "32"))]
+@@ -30,7 +30,7 @@ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+ #[inline]
+ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+     // Emscripten's `makedev` has a 32-bit return value.
+-    Dev::from(unsafe { c::makedev(maj, min) })
++    Dev::from(c::makedev(maj, min))
+ }
+ 
+ #[cfg(not(any(target_os = "android", target_os = "emscripten")))]

--- a/packages/miniserve/build.sh
+++ b/packages/miniserve/build.sh
@@ -2,14 +2,28 @@ TERMUX_PKG_HOMEPAGE=https://github.com/svenstaro/miniserve
 TERMUX_PKG_DESCRIPTION="Tool to serve files and dirs over HTTP"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.21.0"
+TERMUX_PKG_VERSION="0.22.0"
 TERMUX_PKG_SRCURL=https://github.com/svenstaro/miniserve/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=8816d8379321fd3160f57d771b4b743f04bab57dd1ec8d04def0b15a96bde87c
+TERMUX_PKG_SHA256=325f6cde391c468000b1bdcc8455ec2c6950b3c930029187671c536507b185ba
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS=libbz2
 TERMUX_PKG_BUILD_IN_SRC=true
 
 termux_step_pre_configure() {
+	termux_setup_rust
+
+	export CFLAGS="${TARGET_CFLAGS}"
+
+	: "${CARGO_HOME:=$HOME/.cargo}"
+	export CARGO_HOME
+
+	rm -rf $CARGO_HOME/registry/src/github.com-*/rustix-*
+	cargo fetch --target "${CARGO_TARGET_NAME}"
+
+	for d in $CARGO_HOME/registry/src/github.com-*/rustix-*; do
+		patch --silent -p1 -d ${d} < $TERMUX_PKG_BUILDER_DIR/0001-upstream-fix-libc-removing-unsafe-on-makedev.diff || :
+	done
+
 	rm -f Makefile
 }
 

